UPGRADE-5.0.md

```diff
@@ -451,6 +451,8 @@ TwigBundle
  * The default value (`false`) of the `twig.strict_variables` configuration option has been changed to `%kernel.debug%`.
  * The `transchoice` tag and filter have been removed, use the `trans` ones instead with a `%count%` parameter.
  * Removed support for legacy templates directories `src/Resources/views/` and `src/Resources/<BundleName>/views/`, use `templates/` and `templates/bundles/<BundleName>/` instead.
+ * The default value (`twig.controller.exception::showAction`) of the `twig.exception_controller` configuration option has been changed to `null`.
```

Comment:
will the functionality of setting the option be removed as well? if so it should be mentioned that setting the option has no effect.

Reply:
we're setting `null` to enable the new error rendering mechanism (by default), but the option is still usable for custom purposes, if so, you're in control again.

Reply:
although that option probably doesn't make sense anymore.

Reply:
What I have in mind is to also deprecate `\Symfony\Component\HttpKernel\EventListener\ExceptionListener` and instead of triggering a sub-request, we would render a response directly using the ErrorRenderer. But let's discuss that in a separate issue/PR as it's not directly related to this.